rero_ils/modules/documents/views.py

```diff
@@ -716,19 +715,18 @@ def online_holdings(document_pid, viewcode='global'):
 
 
 @blueprint.app_template_filter()
-def subject_format(subject_data, language):
+def subject_format(subject_data, language=None):
     """Format the subject according to the available keys.
 
     :param subject_data: the record subject.
     :param language: current language on interface.
     """
-    try:
-        return SubjectFactory \
-            .create_subject(subject_data) \
-            .render(language=language)
-    except (AttributeError, TypeError) as err:
-        # print(str(err))
-        return 'Subject parsing error !'
+    entity = subject_data['entity']
+    if language is None:
```

Comment:
Normally we don't need this test, because `entity.get(...` will do the job, but it is more clear this way.